### DISCUSSION TITLE
Why should we still using mocha over jest?

### DISCUSSION THREAD

**Discussion by JuanVqz** [rajasegar/alacritty-themes#52] [Q&A]
1. I haven't experienced a lot with jest but feel it's more powerful. 
2. The documentation is better at least I found more examples in the internet.
3. Seems like mocha is old
4. Some stuff didn't work when I wanted to mock the env and the file system or mock a function inside another one.

**Comment by rajasegar** [comment]
@JuanVqz I am in complete agreement for Jest, initially I thought of using Jest only but I wasn't that familiar with it at that point of time, I think now is a good time to take a shot at it.

**Reply by JuanVqz** [reply]
Okay we will have fun with jest. 🚀
